src/components/Message.vue

```diff
@@ -200,6 +202,12 @@ export default {
 
 	&__notsuggested {
 		margin-left: auto;
+		padding-top: 3px;
+		padding-right: 10px;
+		padding-bottom: 3px;
+		display: flex;
+		align-items: center;
+		justify-content: center;
```

Comment:
In general, trying to style only the nonsuggested(a.k.a reply button) is the right way to do it, but in this case the issue is caused by a margin that already exists on the parent class `.reply-buttons`. So try to change the margin-right, to be same as bottom margin :)